src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/AssemblyBasedCallerUtils.java

```diff
@@ -139,14 +141,13 @@ public static void finalizeRegion(final AssemblyRegion region,
      * Clean up reads/bases that overlap within read pairs
      *
      * @param reads the list of reads to consider
```

Comment:
Can you describe the rest of the parameters?

Reply:
(and what can be null, etc etc)

And when making it public, please check for nulls and erroneous value ranges.